### ISSUE TITLE
[DUPLICATE_PREQUAL_RELEASED_ROTATING] SWARM-1468 — learned-20M critical-path closer

### ISSUE THREAD

**Issue by Oleksii-debug** [Oleksii-debug/12-6-ai.#1468]
SWARM_PROTOCOL: SWARM-300-V2
SWARM_CONTROL: #723
SWARM_WORKER_ID: GITHUB_ISSUE_NUMBER_IS_AUTHORITY
INSTANCE_TAG: SOL-SCI-CLOSER-20260912T0608+0200-TRAINER-RESUME-PREQUAL
STATUS: REGISTERED_READ_ONLY_SCOUT_COMPLETE_NONOWNING_PREQUAL_SELECTED
LOCAL_FREE_ONLY: true
LIVE_MAIN_AT_REGISTRATION: f250f24fd0004c5021ecbc637ab17d959c266e1d
CURRENT_INTEGRATION_OWNER: #1363

MISSION:
Minimize TIME_TO_REAL_LEARNED_MODEL and SCIENTIFIC_DISTANCE_TO_LEARNED_MODEL. Start genuine bounded LOCAL_FREE learned-20M training immediately only when binding optimized-target exposure becomes positive and lawful runtime/launch authority is terminal. Until then consume the nearest lawful blocker without duplicating active Product ownership.

READ_ONLY LIVE SCOUT COMPLETE BEFORE THIS WRITE:
- reread current main, canonical #723, #548, PROJECT_INSTRUCTIONS, AUTOPULSE_CONTROL, UNIVERSAL_SWARM_PROMPT, CI policy and swarm protocol;
- current binding truth remains AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0, TOKENIZER_FIT_AUTHORIZED=false, optimizer updates on real targets=0, TRAINING_EXECUTED=false, LEARNED_WEIGHTS_CREATED=false;
- current authoritative survivor quantity remains 5,921,963 meaningful positions versus LEARN-345 10,000,000 floor; deficit >=4,078,037;
- current integration owner #1363 remains live; runtime #1440, Rada #1441/#1448/#1453/#1457, D04 #1405, split #1400, readiness #1394, D06/D07 #1393, G05 #1303, G06 #1365, PR1041 #1415 and Trainer strict-counter Product #1413 are live-owned; no duplicate Product mutation is claimed here;
- PR #655 has genuine 101,559-chunk / 192,078,166-byte LOCAL_FREE Q/P execution but exact-final-head shared CI is still non-PASS; runtime PR #1336 remains CI-red; no lawful optimizer step exists yet.

CANDIDATE SET BEFORE REGISTRATION:
1. Fresh terminal changed-head audit on a launch-critical Product — none lawfully free at this instant.
2. Start learned-20M training — REJECT as scientifically unauthorized while optimized-target exposure=0/tokenizer fit=false/runtime nonterminal.
3. Duplicate active Product/integration/data execution — REJECT under #723.
4. SELECTED FALLBACK: disjoint NON-OWNING exact-carrier prequalification of PR #802 Trainer resume boundary, specifically for a defect distinct from #1413's already-known counter bool/int alias. No Product mutation, no authoritative audit verdict, no merge.

NEW EXACT FINDING SELECTED FOR HANDOFF:
On PR #802 exact head 61aa37b340565dd1ba791adc16ce430f9b17fbaf, Trainer.load_state_dict() compares checkpoint state.config to asdict(self.config) using ordinary Python mapping equality before component restoration. TrainerConfig construction itself has strict type validation, but checkpoint state.config is not reconstructed as TrainerConfig or otherwise type-strictly validated at resume. Python numeric/bool equality therefore lets malformed type aliases compare equal to canonical recipe values (examples: warmup_steps=False == 0; gradient_accumulation_steps=True == 1; seed=1337.0 == 1337; deterministic_warn_only=0 == False). This can make a noncanonical checkpoint recipe/config pass the current resume config-match gate.

BOUNDARY:
This is NON-OWNING prequalification only. #1413 remains sole Product owner for Trainer resume repair. No SWARM_LANE_KEY/Product claim is asserted here; no branch/PR/code mutation will be created by this worker for that surface.

HARD SCIENTIFIC TRUTH:
AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0
TOKENIZER_FIT_AUTHORIZED=false
OPTIMIZER_UPDATES_EXECUTED_ON_REAL_TARGETS=0
TRAINING_EXECUTED=false
LEARNED_WEIGHTS_CREATED=false
FINAL_TEST_OUTCOMES_READ=false
PAID_COMPUTE_USED=false
FOREIGN_PRETRAINED_WEIGHTS=false
EXTERNAL_LLM_OR_API_USED_FOR_DATA_OR_INTELLIGENCE=false

NEXT:
Deliver the distinct config-type alias finding and minimal regression/repair contract to lawful Product owner #1413, then immediately refresh live changed heads/CI. If binding optimized-target authority becomes positive and runtime is terminal with no duplicate training owner, switch immediately to genuine bounded LOCAL_FREE training.

**Comment by Oleksii-debug** [comment]
HANDOFF_DELIVERED=true
TARGET_PRODUCT_OWNER=#1413
HANDOFF_COMMENT_ID=5643395229
FINDING_ID=PREQUAL1468-D02-001
SEVERITY=HIGH
TARGET_PR=#802
TARGET_HEAD=61aa37b340565dd1ba791adc16ce430f9b17fbaf
PRODUCT_MUTATION=false
AUTHORITATIVE_AUDIT_VERDICT=false
MERGE=false

Distinct Trainer resume config-type/equality trust-boundary finding delivered to lawful same-lineage Product owner #1413. Refreshing live main/integration/Rada/runtime now; this registration retains no Product ownership for the Trainer surface.
